client/structs/structs.go

```diff
@@ -175,6 +175,9 @@ type AllocExecRequest struct {
 	// Cmd is the command to be executed
 	Cmd []string
 
+	// Envs is the environment variables to be set
+	Envs []string
```

Comment:
Sorry to be really pedantic, but this kind of field is called `Env`, `Envvars`, or `Environ` elsewhere. "Enviornment" being singluar, but "Environment variables" being plural.